### PR TITLE
remove wrong and unneeded assignment to constant axios

### PR DESCRIPTION
**Pull request by winnyschuster**
axios is defined as a constant, so remove 2nd assignment to it
fixes: #258

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -1279,7 +1278,7 @@ function callUsers(users, text, lang, file, repeats, cb) {
         axios.get(url)
             .then(response => {
                 const body = response && response.data;
-                if (!body || response || response.status !== 200) {
+                if (!body || (response && response.status !== 200)) {
```
Comment (Apollon77) [inline main.js:1281]:
```suggestion
                if (!body || !response || response.status !== 200) {
```

**Comment by maxpd1** [comment]
Hi, thanks for development! Is it possible to turn it into a new release?

**Comment by Apollon77** [comment]
v1.11.0 was released yesterday and should be available in beta channel

**Comment by maxpd1** [comment]
Not sure if problem of my phone or the adapter.
If I get called my phone is ringing and vibrating, but there is no notification of the screen and I can't stop it. I have to telegram app via task manager

**Pull request merged**